pages/sdk/signers/magic.mdx

```diff
@@ -0,0 +1,148 @@
+
+
+import { Tabs, Steps, Callout } from 'nextra/components'
+
+# Magic Signer
+
+In this guide, you will learn how to create a [Magic](https://magic.link/) signer that can be added as a Safe owner and used to initialize any of the kits from the Safe\{Core\} SDK.
+
+Check out the [Safe Signers demo app](https://github.com/5afe/safe-signers) on GitHub to follow along this guide with the completed code.
+
+## Prerequisites
+
+- [Node.js and npm](https://docs.npmjs.com/downloading-and-installing-node-js-and-npm).
+- A [Magic account](https://dashboard.magic.link) and an API key.
+
+## Install dependencies
+
+{/* <!-- vale off --> */}
+
+<Tabs items={['npm', 'yarn', 'pnpm']}>
+  <Tabs.Tab>
+    ```bash
+    npm install magic-sdk
+    ```
+  </Tabs.Tab>
+  <Tabs.Tab>
+    ```bash
+    yarn add magic-sdk
+    ```
+  </Tabs.Tab>
+  <Tabs.Tab>
+    ```bash
+    pnpm add magic-sdk
+    ```
+  </Tabs.Tab>
+</Tabs>
+
+{/* <!-- vale on --> */}
+
+## Steps
+
+<Steps>
+
+  ### Imports
+
+  Here are all the necessary imports for this guide.
+
+  {/* <!-- vale off --> */}
+
+  ```typescript
+  import { Magic } from 'magic-sdk'
+  ```
+
+  {/* <!-- vale on --> */}
+
+  ### Get the API Key
+
+  Check Magic documentation on how to create a new project in their [dashboard](https://dashboard.magic.link) and [get the API key.](https://magic.link/docs/home/quickstart/cli#add-your-api-key)
```

Comment:
We are mixing the "we" and "you" addressing in the guide. Let's use the direct "you" (instructional) tone throughout.